lib/ebaze_web/templates/auction/show.html.eex

```diff
@@ -0,0 +1,43 @@
+<h1>Show Auction</h1>
+
+<ul>
+
+  <li>
+    <strong>Name:</strong>
+    <%= @auction.name %>
+  </li>
+
+  <li>
+    <strong>Description:</strong>
+    <%= @auction.description %>
+  </li>
+
+  <li>
+    <strong>Sold status:</strong>
+    <%= @auction.sold_status %>
+  </li>
+
+  <li>
+    <strong>Initial price:</strong>
+    <%= @auction.initial_price %>
+  </li>
+
+  <li>
+    <strong>Start time:</strong>
+    <%= @auction.start_time %>
+  </li>
+
+  <li>
+    <strong>End time:</strong>
+    <%= @auction.end_time %>
+  </li>
+
+  <li>
+    <strong>Photo url:</strong>
+    <%= @auction.photo_url %>
```

Comment:
Could you put this in `img` tag so it actually displays? (maybe make a card for that)

Reply:
e91e6a2172fb6cbae80310a4d34a79efd5b4bb65